class.js

```diff
@@ -1,20 +1,20 @@
-class Foo
-{
-color='negro';peso   =    34;
-  ver()
-  {
-    console.info(`Color: ${this.color}. Peso: ${this.peso}`
-    );
-// foo = 123;
-/*
-// Prueba
-test() {
-  console.info('Test');
+class Foo {
+  color = 'negro';
+  peso = 34;
+  ver () {
+    console.info (`Color: ${this.color}. Peso: ${this.peso}`);
+  }
+  test() {
+      console.info('Test');
+  }
 }
-*/
-}
-   }    class Bar   {
-  miMetodo(){if(this.imprimir){console.info('Imprimir!');}}
+
+class Bar {
+  miMetodo () {
```

Comment:
No se deja un espacio entre el nombre del método y los `()`.

El espacio se deja para las funciones especiales como `if`, `for`, `while`, etc...
```suggestion
  miMetodo() {
```